### PR TITLE
Add Cody billing user definitions and update existing docs

### PR DESCRIPTION
**Pull request by dadlerj**
## Pull Request approval

Although pull request approval is not enforced for this repository in order to reduce friction, merging without a review will generate a ticket for the docs team to review your changes. So if possible, have your pull request approved before merging.

### CODE REVIEW

File: docs/admin/pricing.mdx [hunk 1]
```diff
@@ -2,10 +2,40 @@
...
+### Billable active users
 
+_Note: this only applies to Enterprise Cody contracts. Cody Pro users pay for a seat every month, regardless of usage._
+
+A billable active Cody user is one who is signed in to their Enterprise Sourcegraph account and actively
+interacts with Cody (e.g. they see suggested autocompletions, they run commands or chat with
+Cody, they start new discussions, clear chat history, or copy text from chats, they change Cody
+settings, and more). 
+
+Simply having Cody installed is not enough to be considered a billable user.
```
Comment (aramaraju) [inline docs/admin/pricing.mdx:16]: Should we clarify this as `Cody installed and authenticated against the instance` to be extra clear here?

**Review by aramaraju — APPROVED** [review]
Minor nitpick, otherwise LGTM

File: docs/admin/pricing.mdx (hunk 1 above)
Comment (dadlerj) [reply docs/admin/pricing.mdx]: Yep good call. I adjusted the wording slightly to `Cody installed and signed in`

**Review by taras-yemets — APPROVED** [review]

File: docs/admin/pricing.mdx [hunk 2]
```diff
@@ -2,10 +2,40 @@
 
 Visit the [pricing page](https://about.sourcegraph.com/pricing) for up-to-date pricing. 
 
-## How are active users calculated?
+## How are active users calculated for **Sourcegraph Cody**?
 
-Any user who accesses Sourcegraph in a given month is considered a monthly active user (MAU). This includes but is not limited to:
+### Billable active users
 
+_Note: this only applies to Enterprise Cody contracts. Cody Pro users pay for a seat every month, regardless of usage._
```
Comment (MaedahBatool) [inline docs/admin/pricing.mdx:9]:
```suggestion
<Callout type="note">This only applies to Enterprise Cody contracts. Cody Pro users pay for a seat every month, regardless of usage.</Callout>
```

Can we please use these callouts.

**Review by MaedahBatool — APPROVED** [review]
Looks good. If we are ready let's point this branch to the working branch for 5.5 release.

**Pull request merged**